RecoLocalTracker/Phase2TrackerRecHits/python/Phase2TrackerRecHits_cfi.py

```diff
@@ -3,7 +3,5 @@
 # RecHits options
 siPhase2RecHits = cms.EDProducer("Phase2TrackerRecHits",
   src = cms.InputTag("siPhase2Clusters"),
-  Phase2StripCPE = cms.ESInputTag("phase2StripCPEESProducer", "Phase2StripCPEDummy")
+  Phase2StripCPE = cms.ESInputTag("phase2StripCPEESProducer", "Phase2StripCPETrivial")
```

Comment:
yes, that's what I meant.
I understood from Monday slides that Phase2StripCPETrivial is more wrong and we need something more correct.